src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ThemeServiceTests.cs

```diff
@@ -576,38 +912,43 @@ public async Task DeleteUITestThemes_DisallowedByConfiguration()
 
             await using (var context = InMemoryApplicationDbContext(contextId))
             {
-                var topicService = new Mock<ITopicService>(Strict);
-
-                var service = SetupThemeService(
-                    context,
-                    topicService: topicService.Object,
-                    enableThemeDeletion: false);
+                var service = SetupThemeService(context, enableThemeDeletion: false);
 
                 var result = await service.DeleteUITestThemes();
-                VerifyAllMocks(topicService);
                 result.AssertForbidden();
 
                 Assert.Equal(1, await context.Themes.CountAsync());
             }
         }
 
         private static ThemeService SetupThemeService(
-            ContentDbContext context,
+            ContentDbContext? contentContext = null,
+            StatisticsDbContext? statisticsContext = null,
             IMapper? mapper = null,
-            IPersistenceHelper<ContentDbContext>? persistenceHelper = null,
             IUserService? userService = null,
-            ITopicService? topicService = null,
+            IBlobCacheService? cacheService = null,
+            IMethodologyService? methodologyService = null,
+            IReleaseSubjectRepository? releaseSubjectRepository = null,
+            IReleaseFileService? releaseFileService = null,
+            IReleaseDataFileService? releaseDataFileService = null,
+            IReleasePublishingStatusRepository? releasePublishingStatusRepository = null,
             IPublishingService? publishingService = null,
             bool enableThemeDeletion = true)
         {
             return new ThemeService(
                 new AppOptions { EnableThemeDeletion = enableThemeDeletion }.ToOptionsWrapper(),
-                context,
+                contentContext ?? new Mock<ContentDbContext>().Object,
+                statisticsContext ?? new Mock<StatisticsDbContext>().Object,
                 mapper ?? AdminMapper(),
-                persistenceHelper ?? new PersistenceHelper<ContentDbContext>(context),
+                new PersistenceHelper<ContentDbContext>(contentContext!),
```

Comment:
Not a big deal, but I think this might fail if no `contentContext` is passed to this method. In other situations like this we've done `var contentDbContext = contentContext ?? new Mock[...]` before constructing the service itself.